ux/src/main/java/pro/streem/ar/sceneform/ux/BaseArFragment.java

```diff
@@ -345,16 +345,31 @@ protected void setCanRequestDangerousPermissions(Boolean canRequestDangerousPerm
         this.canRequestDangerousPermissions = canRequestDangerousPermissions;
     }
 
+    protected boolean supportsPictureInPicture = false;
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        if (supportsPictureInPicture) {
```

Comment:
My thought was to only change lifecycle operations in the Sceneform SDK if the user of the SDK explicitly requests it, to avoid potentially breaking things for existing users. Streem may want AR features to continue working in the system PiP, but other users may not.